raiden-dapp/tests/unit/store/notifications/mutations.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { mutations } from '@/store/notifications/mutations';
+import { TestData } from '../../data/mock-data';
+import {
+  NotificationPayload,
+  NotificationsState,
+} from '@/store/notifications/types';
+
+describe('notifications store mutations', () => {
+  const notification = TestData.notifications;
+
+  const createNotificationState = (
+    newNotifications = false
+  ): NotificationsState => {
+    return {
+      notifications: { 1: notification },
+      newNotifications,
+    };
+  };
+
+  test('can delete notification', () => {
+    const state = createNotificationState();
+    const id = TestData.notifications.id;
```

Comment:
You stored it extra in a variable before. Why not using it now? :thinking:

Reply:
I overlooked that, fixed it now though.